sys/include/net/gnrc/netif/hdr.h

```diff
@@ -24,6 +24,7 @@
 #include <string.h>
 #include <stdint.h>
 
+#include "net/gnrc/netif.h"
```

Comment:
The functions got moved to netif/internal.h in the PR that adds them.

Reply:
This was addressed in https://github.com/RIOT-OS/RIOT/pull/10525. I just did not rebase yet.